tests/identify_disks_tests.c

```diff
@@ -355,6 +355,18 @@ static void test_identify_disks_no_sys_class_nvme_present(void **state)
     assert_string_equal(capture_stdout(), "");
 }
 
+static void test_identify_disks_sys_class_nvme_empty(void **state)
+{
+    (void)state; // Unused parameter
+
+    struct context ctx = {.output_format = PLAIN};
+    int result = identify_disks(&ctx);
+
+    assert_int_equal(result, 0);
+    assert_string_not_equal(capture_stderr(), "no NVMe devices in");
```

Comment:
This test case should already be covered in
`{"no namespaces", NULL, "", ""},` in test cases below.

I'm ok with keeping here, but this check will always pass and should be:
assert_string_equal(capture_stderr(), "");
or
assert_null(strstr(stderr_output, "no NVMe devices in"));